worlds/dark_souls_3/Options.py

```diff
@@ -209,12 +209,14 @@ class AutoEquipOption(Toggle):
 
 
 class LockEquipOption(Toggle):
-    """Lock the equipment slots so you cannot change your armor or your left/right weapons. Works great with the Auto-equip option."""
+    """Lock the equipment slots so you cannot change your armor or your left/right weapons.
+    Works great with the Auto-equip option."""
```

Comment:
I think we should actively avoid reformatting these option docs, since the most important place they're surfaced is the web view where newlines are preserved exactly as-is.